packages/env-utilities/README.md

```diff
@@ -0,0 +1,56 @@
+# `@skyra/env-utilities`
+
+Functional utilities for reading and parsing environmental variables, based on [Skyra](https://skyra.pw)'s internal tools.
+
+## Usage
+
+### Setup
+
+To setup dotenv, you use the `setup` function exported by the package:
+
+```typescript
+import { setup } from '@skyra/env-utilities';
+
+setup(import.meta.url);
```

Comment:
Specify what the default path is when no options are passed (presumably root dir)
And specify that this will set the dir to the folder where this file is.